localstack/services/dynamodb/provider.py

```diff
@@ -545,7 +553,7 @@ def describe_table(self, context: RequestContext, table_name: TableName) -> Desc
                     "TableClass": table_definitions["TableClass"]
                 }
 
-        return result
+        return DescribeTableOutput(Table=select_from_typed_dict(TableDescription, result["Table"]))
```

Comment:
nit: in a few lines above we're hedging with `result.get("Table", {})`, whereas here we're using `result["Table"]`.  Not sure if there can be a case where Table would not be present (`_forward_request(..)` would raise if the table doesn't exist), so perhaps we could change the other lines to use `result["Table"]` as well.. 👍 (just cosmetics, not critical at all..)